packages/components/src/components/tabs/tabs.js

```diff
@@ -114,21 +114,40 @@ class Tab extends ContentSwitcher {
   _handleKeyDown(event) {
     const triggerNode = eventMatches(event, this.options.selectorTrigger);
     if (triggerNode) {
+      // enter
       if (event.which === 13) {
```

Comment:
```suggestion
      if (match(event, keys.Enter)) {
```

Reply:
do you mean I should directly import the `match` function from the react package?